typescript/packages/plugins/uniswap/README.md

```diff
@@ -0,0 +1,34 @@
+# Goat Uniswap Plugin 🐐 - TypeScript
+
+Uniswap plugin for [Goat 🐐](https://ohmygoat.dev). Allows you to create tools for interacting with Uniswap.
+
+## Installation
+```
+npm install @goat-sdk/plugin-uniswap
+```
+
+## Usage
+
+```typescript
+import { uniswap } from "@goat-sdk/plugin-uniswap";
+
+
+const plugin = uniswap({
+    baseUrl: process.env.UNISWAP_BASE_URL as string,
+    apiKey: process.env.UNISWAP_API_KEY as string,
```

Comment:
Where does one get a Uniswap API key? It'd be great to link to that on the readme

Reply:
Ah I included the link to the landing page and form in the *vercel ai* example I created - shall I add it here too?